make/autoconf/basic_tools.m4

```diff
@@ -106,9 +107,6 @@ AC_DEFUN_ONCE([BASIC_SETUP_TOOLS],
   UTIL_LOOKUP_PROGS(READLINK, greadlink readlink)
   UTIL_LOOKUP_PROGS(WHOAMI, whoami)
 
-  # Tools only needed on some platforms
```

Comment:
Comment still relevant? Or is it needed on all platforms?

Reply:
No, so I removed it.

Reply:
Ah, sorry, now I realize what you meant (perhaps). I moved the line to a block of lookups that is already preceded by the very same comment. It is still true that lsb_release is only needed on some platforms.